blazer/app/models/audit.rb

```diff
@@ -7,7 +7,7 @@ class Audit < Record
     private
 
     def log_user
-      logger.info "Audit #{user.email} ran '#{query.name}' query: '#{query.statement}'"
```

Comment:
`query` does not exist for queries run while editing. But `statement` is there in either case.